dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe

```diff
@@ -15,26 +15,36 @@ PATCHES="libmysqlclient-6.1.6.patchset"
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersion="18.3.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
-	libmysqlclient$secondaryArchSuffix = 18.3.0 compat >= 18.3
-	cmd:my_print_defaults$secondaryArchSuffix
-	cmd:mysql_config$secondaryArchSuffix
-	cmd:perror$secondaryArchSuffix
-	lib:libmysqlclient$secondaryArchSuffix = 18.3.0 compat >= 18.3
-	lib:libmysqlclient_r$secondaryArchSuffix = 18.3.0 compat >= 18.3
+	libmysqlclient$secondaryArchSuffix = $portVersion
+	lib:libmysqlclient$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
-	libmysqlclient${secondaryArchSuffix}_devel = 18.3.0 compat >= 18.3
-	devel:libmysqlclient$secondaryArchSuffix = 18.3.0 compat >= 18.3
-	devel:libmysqlclient_r$secondaryArchSuffix = 18.3.0 compat >= 18.3
+	libmysqlclient${secondaryArchSuffix}_devel = $portVersion
+	cmd:mysql_config$secondaryArchSuffix
+	devel:libmysqlclient$secondaryArchSuffix = $libVersionCompat
+	devel:libmysqlclient_r$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES_devel="
-	libmysqlclient$secondaryArchSuffix == 18.3.0
+	libmysqlclient$secondaryArchSuffix == $portVersion base
+	"
+
+PROVIDES_tools="
+	libmysqlclient${secondaryArchSuffix}_tools = $portVersion
+	cmd:my_print_defaults$secondaryArchSuffix
```

Comment:
Not sure if this should be in tools or in the devel package, comments welcome :)

Reply:
no, only mysql_config should be in _devel.